scrapli/transport/plugins/telnet/transport.py

```diff
@@ -119,21 +148,11 @@ def _handle_control_chars(self) -> None:
 
         control_buf = b""
 
-        original_socket_timeout = self._base_transport_args.timeout_socket
-        self._set_socket_timeout(self._base_transport_args.timeout_socket / 4)
```

Comment:
I removed the timeout stuff here because I think if we left it that may be problematic as the socket timeout would be very low basically forever since we were never setting it back to "normal"... *but* to be fair I guess I wasn't setting it back to normal (whatever the default is or what the user provides) before either...